### ISSUE TITLE
Rsrp/Rsrq Signal Information Not Updating with CellSignalStrengths changing 

### ISSUE THREAD

**Issue by ragiovan** [xamarin/xamarin-android#8205]
### Android application type

Classic Xamarin.Android (MonoAndroid12.0, etc.)

### Affected platform version

VS 2022

### Description

Using Android.Telephony and a StateChange listener, the signal strength listener is called whenever rsrp changes (When it happens can be verified going into the sim status changes and looking at rsrp) . However, when looking through the actual Registered LTE signals LTE rsrp/rsrq is not changing at all.

### Steps to Reproduce

Standalone Solution - in .net maui, using Android API: https://github.com/ragiovan/cellularstrengthsNotChanging

Open tst solution. Connect android phone and start waving it all around. Put in a faraday cage if you want. The rsrp/rsrq received will never change.

### Did you find any workaround?

No

**Comment by jpobst** [comment]
Ultimately, all Xamarin.Android does is provide generated wrappers that allow you to call Google’s Android API. We do not have much expertise in how to use those API’s properly. 

However, it appears to work as I would assume when running in the emulator:

![image](https://github.com/xamarin/xamarin-android/assets/179295/0a20efa8-b9e4-4984-8c09-dde9aa50e02e)

Changing the "Signal Strength" in the emulator options hits your breakpoint.

**Comment by ragiovan** [comment]
I would expect it would work when changing "Signal Strength" in the emulator. That's because you are triggering the change.

My claim is that "Signal Strength" events from the API on a real phone is only changed once and only once, at the start of the program. The API is never changed again.

**Comment by jpobst** [comment]
Agreed, the point of showing the emulator is simply to demonstrate that I do not believe this is an issue with our wrapper.  When the Android system calls the `onSignalStrengthsChanged`, we do indeed raise the proper event.

As for when or how Android chooses to call that API, I think that is a question for Google or the device manufacturer.
